pgjdbc/src/main/java/org/postgresql/jdbc/BooleanTypeUtil.java

```diff
@@ -74,20 +92,20 @@ private static boolean from(final Character charval) throws PSQLException {
         || 'n' == charval || 'N' == charval) {
       return false;
     }
-    throw cannotCoerceException("Character", charval.toString());
+    throw cannotCoerceException(fromType, charval.toString());
   }
 
-  private static boolean from(final Number numval, final String className) throws PSQLException {
+  private static boolean fromNumber(final Number numval, final String fromType) throws PSQLException {
     // Handles BigDecimal, Byte, Short, Integer, Long Float, Double
     // based on the widening primitive conversions.
-    double value = numval.doubleValue();
-    if (value == 1.0d) {
+    final double value = numval.doubleValue();
+    if (1.0D == value) {
```

Comment:
-1 In general I prefer not to change style unless we change the entire style (which I am not endorsing)